.github/workflows/CITest.yml

```diff
@@ -34,41 +34,53 @@ jobs:
               os: ubuntu-22.04,
               arch: x64,
               build-system: 'make',
+              diet-build: 'OFF',
+              enable-asan: 'OFF'
+            }
+          - {
+              name: 'ubuntu-latest x64 cmake',
+              os: ubuntu-latest,
```

Comment:
ubuntu-latest is Ubuntu 22.04 according to https://github.com/actions/runner-images. How about we only run cmake on ubuntu-24.04.